### PR TITLE
Update with-office-365.md

### PR DESCRIPTION
**Pull request by lindspea**
https://github.com/MicrosoftDocs/OfficeDocs-SkypeForBusiness/issues/2668
Deleted note and added new note and command.

### CODE REVIEW

**Comment by scanum** [comment]
@nenonix @JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Teams/room-systems/with-office-365.md [hunk 1]
```diff
@@ -177,17 +177,16 @@ For details on Skype for Business Online Plans, see the [Skype for Business Onli
...
 
    Obtain the RegistrarPool information from the new user account being setup, as shown in this example:
 
     ``` Powershell
     Get-CsOnlineUser -Identity $rm | Select -Expand RegistrarPool
     ```
-
     > [!NOTE]
-    > New user accounts might not be created on the same registrar pool as existing user accounts in the tenant. The command above will prevent errors in account setup due to this situation.
+    > All new Microsoft Teams Room accounts must be created in the same RegistrarPool.
```
Comment (JohanFreelancer9) [inline Teams/room-systems/with-office-365.md]:
```suggestion
    > All new Microsoft Teams Rooms accounts must be created in the same RegistrarPool.
```

**Comment by scanum** [comment]
@LanaChin PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Comment by LanaChin** [comment]
Reached out to PM to review updates.

**Comment by scanum** [comment]
@LanaChin Any updates from PM?

**Comment by jambirk** [comment]
Added Sohail and Greg. Approval needed from one is enough to proceed.

**Comment by scanum** [comment]
@sohailta PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Comment by scanum** [comment]
@jambirk Hi James, wondering if there is an update on this PR? Any response from Sohail? Thanks!

**Comment by jambirk** [comment]
@aageorge can you comment?

**Comment by jambirk** [comment]
The Product development group has declined to make this change.

**Pull request closed without merging**